### ISSUE TITLE
dev center sitemap.xml

### ISSUE THREAD

**Issue by mkiser** [algorithmiaio/dev-center#53]
Not sure what the best way of doing this is, so here are two options:

1) Generate a sitemap.xml for the dev center content and include it in https://algorithmia.com/robots.txt (ie. adding a line Sitemap: https://algorithmia.com/developers/sitemap.xml)

2) Add/update all the pages to the current sitemap. I think this is preferred if it can be automated/always up to date. 

In either event, we want to have a comprehensive sitemap

**Comment by peckjon** [comment]
For as long as sample-apps is a separate project with its own deploy-flow, (1) would give sample-app-devs more manual control over what's visible.  But also require manual action.

Which reminds me that I don't have a way to remove a sample app once it has been deployed... if I did, (2) would not be a problem.

**Comment by anowell** [comment]
I've been debating the same two options. Option 1 is trivial, as jekyll already generates the sitemap. I do think I have an idea of how to accomplish Option 2, so I'm gonna at least start down that to get a sense of how deep that rabbit hole goes. I'm not sure how the 'sample-apps' repo plays into this @peckjon as this only applies to content hosted under the main domain (it would apply to a future [uncommitted] demo site hosted under the main domain).

Implementation details: I'm thinking the 12-hour cache miss that regenerates the sitemap, can just read /developers/sitemap.xml, parse out all the entries, and add them (in [this code](https://github.com/algorithmiaio/algorithmia/blob/1e609b41e56d165123a7eabaedc26c2024d2fb4a/app/util/Sitemap.scala))... it "sounds" straight forward.

**Comment by anowell** [comment]
This is now implemented in the `devcenter_sitemap` branch of the webserver. It's worth mentioning, the webserver sets `loc`, `changefreq`, and `priority`, jekyll currently only sets `loc`, and `lastmod` on their pages. If we really want priority for devcenter pages, we'll be forking/modifying the jekyll-sitemap plugin to add it back. The upstream developers explicitly removed priority from jekyll sitemaps: see https://github.com/jekyll/jekyll-sitemap/issues/152

Note: I don't want to merge `devcenter_sitemap` until after we've flipped a switch (otherwise we'll start getting pages indexed before we're ready to link to them). I want to leave this issue open so it doesn't get forgotten, but it is NOT blocking the devcenter release.
